spec/build_spec.lua

```diff
@@ -132,21 +129,21 @@ describe("LuaRocks build tests #integration", function()
    end)
 
    describe("LuaRocks build - basic builds", function()
-      it("LuaRocks build luadoc", function()
+      pending("LuaRocks build luadoc", function()
          assert.is_true(run.luarocks_bool("build luadoc"))
       end)
       
-      it("LuaRocks build luacov diff version", function()
+      pending("LuaRocks build luacov diff version", function()
          assert.is_true(run.luarocks_bool("build luacov 0.11.0-1"))
          assert.is.truthy(lfs.attributes(testing_paths.testing_sys_rocks .. "/luacov/0.11.0-1/luacov-0.11.0-1.rockspec"))
       end)
       
-      it("LuaRocks build command stdlib", function()
+      pending("LuaRocks build command stdlib", function()
```

Comment:
This kind of test is probably a better candidate for removal because it runs the same command as another test, only using a different existing rock which happens to trigger a different build path (I think this one was in because stdlib used to use `build.type = "command"`...)